projects/plugins/jetpack/extensions/blocks/premium-content/edit.js

```diff
@@ -332,21 +332,21 @@ function Edit( props ) {
 				<ToolbarGroup>
```

Comment:
This diff looks messier than it is, essentially I cut the Subscriber view and pasted it above the Guest View